gourmet/backends/db.py

```diff
@@ -452,6 +452,8 @@ def backup_db (self):
         print('You can use it to restore if something ugly happens.')
         shutil.copy(self.filename,backup_file_name) # Make a backup...
         import gourmet.gtk_extras.dialog_extras as de
+        from gi import require_version
```

Comment:
I would prefer not to have conditional imports

Reply:
> Hey, thanks for taking the time to do this tedious work!
> I have a little comment, which might be a nitpick, but since you're cleaning the imports, it may as well be addressed :)

I agree, I prefer not to embed imports within the code but rather place them at the top of the module where they are visible to all. This does introduce startup overhead but it is only paid once. I am hesitant in doing 'while you at it, will you do x" as it can complicate the PR and  open up whole new unrelated can of worms.

Moving the embedded imports to the top of the file is not so little. Sometimes imports are embedded within functions to prevent circular dependencies. 

As you can tell, I am pushing back on this. Instead, I propose opening a ticket to clean up the imports.
- Move embedded imports to the top of the module
- Split multi-line imports (PEP8 "single line per import")
- Group `from module import` on a single line and in alphabetical order :-)
- Remove unused imports
- Group imports into the following order (when possible):
  - standard libraries imports
  - third party imports
  - local application specific imports
- I'm sure there are more import stuff that can be done

I suggest tackling this after we have stabilised the migration. Thoughts?

Reply:
I've moved 'gi.repository' imports that were buried within methods and functions to the top of the modules. However, I did not move 'gi.repository' imports within __main__ entry points as these needs to be cleaned up. Many __main__ are used as test entry points to the module and are obsolete/